h2o-parsers/h2o-parquet-parser/src/test/java/water/parser/parquet/ParseTestParquet.java

```diff
@@ -348,6 +379,39 @@ static File generateParquetFile(File parentDir, String filename, int nrows, Date
     return f;
   }
 
+  static File generateLargeStringParquet(File parentDir, String filename) throws IOException {
+    File f = new File(parentDir, filename);
+
+    Configuration conf = new Configuration();
+    MessageType schema = parseMessageType(
+        "message test { "
+            + "required BINARY string_field; "
+            + "} ");
+    GroupWriteSupport.setSchema(schema, conf);
+    SimpleGroupFactory fact = new SimpleGroupFactory(schema);
+    ParquetWriter<Group> writer = new ParquetWriter<Group>(new Path(f.getPath()), new GroupWriteSupport(),
+        UNCOMPRESSED,
+        262144, 1024, 512, true, false, ParquetProperties.WriterVersion.PARQUET_2_0, conf);
+    try {
+      //This test may fail on Java 9, as it will use 1 byte per char and sometimes 2 bytes
+      Binary binary = Binary.fromString(fillString(Integer.MAX_VALUE / 4, 'c'));
```

Comment:
I would prefer if the test used lower memory and can run locally without issues for everybody